.github/workflows/maven-verify.yml

```diff
@@ -29,6 +29,6 @@ jobs:
       maven-args: '-D"maven.test.redirectTestOutputToFile=false"'
       maven-matrix: '[ "3.9.9", "4.0.0-rc-3" ]'
       maven4-enabled: true
-      matrix-exclude: '[ {"jdk": "8"} ]'
+      matrix-exclude: '[ {"jdk": "21"} ]'
```

Comment:
Why you want to exclude build on JDK 21 ... when you want migrate to 21?

Reply:
sorry im replacing 8 with 21 in hope with bump but this makes no sense yes. sorry will revert